app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] [OPTIONAL] Please use the callback `after_save` to call the `update_comments_counter`, so the method is called automatically every time a new post has been saved to the database, then make the `update_comments_counter` method private.

Reply:
@gemmen29 thank you for the feedback. It is very helpful